tests/test_kerascv.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+import json
+import keras_cv
+import keras
+import numpy as np
+
+class TestKerasCV(unittest.TestCase):
+    def test_inference(self):
+        classifier = keras_cv.models.ImageClassifier.from_preset(
+            'efficientnetv2_b0_imagenet_classifier',
+            load_weights=False, # load randomly initialized model from preset architecture with weights
```

Comment:
Smart way to avoid having to download weights which can't be done in these isolated tests :)